test/edu/stanford/dlss/wowza/TestSulWowza.java

```diff
@@ -733,6 +735,45 @@ public void verifyTokenAgainstStacksService_validUrl()
     }
 
     @Test
+    public void verifyTokenAgainstStacksService_filenameNeedsEncoding()
+    {
+        WMSProperties mockProperties = mock(WMSProperties.class);
+        when(mockProperties.getPropertyStr(anyString(), anyString())).thenReturn(SulWowza.DEFAULT_STACKS_TOKEN_VERIFICATION_BASEURL);
+        IApplicationInstance appInstanceMock = mock(IApplicationInstance.class);
+        when(appInstanceMock.getProperties()).thenReturn(mockProperties);
+        testModule.onAppStart(appInstanceMock);
+        String druid = "oo000oo0000";
+        String filename = "file name with spaces; and an unusual char @ then end.ext!";
```

Comment:
I might use a filename more like "(special chars) '.|-+!,[]"   and add in a utf-8 char for good measure.  This is based on https://github.com/sul-dlss/media/wiki/Examples-of-files-deposited-by-ETD-users,-special-characters-and-media-files